### DISCUSSION TITLE
Help me defined Array of ObjectId relationship?

### DISCUSSION THREAD

**Discussion by masterbater** [prisma/prisma#13380] [Q&A]
```
model Manager{
  id                     String   @id @default(auto()) @map("_id") @db.ObjectId
  name               String
  assignedWorker String[]              @db.ObjectId
}
model Worker{
  id                     String   @id @default(auto()) @map("_id") @db.ObjectId
  name               String
}
```
Example Data
```
 {id: "60d5922d00581b8f0062e3a8", name: "Manager Gwapo", assignedWorker: ["60d5922d00581b8f0062e3a9", "60d5922d00581b8f0062e3a5"]}
 
 [ {id: "60d5922d00581b8f0062e3a9", name: "Worker 1"},  {id: "60d5922d00581b8f0062e3a5", name: "Worker 2"}]
```

**Comment by nurul3101** [comment]
Hey @masterbater 👋 

Could you please elaborate on what you are trying to achieve?

**Reply by vladi-stevanovic** [reply]
👋 Hello @masterbater  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
